src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioIntegrationTestUtilities.csproj

```diff
@@ -56,6 +56,11 @@
     <PackageReference Include="Microsoft.VisualStudio.VsInteractiveWindow" Version="$(MicrosoftVisualStudioVsInteractiveWindowVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.ComponentModelHost" Version="$(MicrosoftVisualStudioComponentModelHostVersion)" />
     <PackageReference Include="System.Collections.Immutable" Version="$(SystemCollectionsImmutableVersion)" />
+    <!-- 
+      Required to avoid double write of System.Threading.Tasks.Dataflow.dll caused by both 
+      Microsoft.VisualStudio.ProjectSystem and Microsoft.VisualStudio.Composition containing it.
```

Comment:
Is there a bug tracking this we can reference here?

Reply:
Who should get the bug though? @AArnott agreed Microsoft.VisualStudio.ProjectSystem should move to System.Threading.Tasks.Dataflow.dll instead of obsolete Microsoft.Tpl.Dataflow. I can file such bug against Microsoft.VisualStudio.ProjectSystem.
Also Nuget has related, but Won't fixed bug https://github.com/NuGet/Home/issues/329

Reply:
Filed https://devdiv.visualstudio.com/DevDiv/_workitems/edit/547065 against CPS to move from Microsoft.Tpl.Dataflow. I will reference it here.